### PR TITLE
Fix edit content button

### PR DESCRIPTION
**Pull request by mhyndle**
Edit content button was lacking the content passed to attribute.

### CODE REVIEW

**Comment by mhyndle** [comment]
ping @dpobel @StephaneDiot @yannickroger

**Comment by dpobel** [comment]
no unit test ?

**Comment by mhyndle** [comment]
tests added

File: Tests/js/views/assets/ez-actionbarview-tests.js [hunk 1]
```diff
@@ -37,49 +37,60 @@ YUI.add('ez-actionbarview-tests', function (Y) {
             this.view.destroy();
         },
 
-        "Should instantiate createContentActionView with contentType": function () {
-            var createContentActionFound = false;
+        _actionAssertions: function (testedActionId) {
```
Comment (dpobel) [inline Tests/js/views/assets/ez-actionbarview-tests.js]: bad name, method should start with a verb, so here `_isActionCreated` or something like that.

File: Tests/js/views/assets/ez-actionbarview-tests.js [hunk 2]
```diff
@@ -37,49 +37,60 @@ YUI.add('ez-actionbarview-tests', function (Y) {
...
-                    createContentActionFound = true;
-                    Assert.areSame(
-                        this.view.get('contentType'),
-                        this.view.get('actionsList')[i].get('contentType'),
-                        'The contentType should have been set to createContentActionView'
-                    );
+                if (this.view.get('actionsList')[i].get('actionId') === testedActionId){
+                    actionFound = true;
+
+                    switch ( testedActionId ) {
```
Comment (dpobel) [inline Tests/js/views/assets/ez-actionbarview-tests.js]: not generic, instead the assert method should take in second argument the attributes if should check, something like:
```js
this._isActionCreated('translate', {
    location: this.view.get('location'),
    content: this.view.get('content'),
});
```

File: Tests/js/views/assets/ez-actionbarview-tests.js [hunk 3]
```diff
@@ -37,49 +37,49 @@ YUI.add('ez-actionbarview-tests', function (Y) {
...
-                    createContentActionFound = true;
-                    Assert.areSame(
-                        this.view.get('contentType'),
-                        this.view.get('actionsList')[i].get('contentType'),
-                        'The contentType should have been set to createContentActionView'
-                    );
+                if (this.view.get('actionsList')[i].get('actionId') === actionId){
+                    actionFound = true;
+
+                    /*jshint loopfunc: true */
```
Comment (mhyndle) [inline Tests/js/views/assets/ez-actionbarview-tests.js]: I put the directive to ignore loopfunc tests on it as creating external method for just checking the assertion seems too complicated for me.

**Comment by StephaneDiot** [comment]
+1

File: Tests/js/views/assets/ez-actionbarview-tests.js (hunk 3 above)
Comment (dpobel) [inline Tests/js/views/assets/ez-actionbarview-tests.js]: you can also just find the action view in the loop (btw, Y.Array.find would be a bit nicer than this for loop) and then iterate to compare the attribute.

Comment (mhyndle) [inline Tests/js/views/assets/ez-actionbarview-tests.js]: Oh right, I have separated iteration through actions and iteration through attributes.

**Comment by mhyndle** [comment]
travis test fails, however not on js part ;)

**Comment by dpobel** [comment]
+1

**Pull request merged**